style/objective_c/README.md

```diff
@@ -11,8 +11,8 @@ Objective-C
 * Order `@class` directives alphabetically.
 * Order `@property` modifiers: memory management, atomicity, writability.
 * Leave out `@property` modifiers unless needed, `nonatomic` is the only one
-  needed in most cases except connecting views with IB in which case `weak` may
-  also be needed.
+  needed in most cases.
+* Avoid `weak` references to IBOutlets.
```

Comment:
Yes. The only reason I went for that is that we get `weak` by default. Fine with either.